201-virtual-wan-with-all-gateways/README.md

```diff
@@ -0,0 +1,81 @@
+
+> [!WARNING]
+> Default values for parameters ***Hub1_PublicCertificateDataForP2S*** and ***Hub2_PublicCertificateDataForP2S***  (Point-to-Site (P2S) configuration) have been provided only as a sample to complete quickly the vWAN deployment procedure. After the deployment is completed, you should generate your own certificates and change each Point-to-Site (P2S) Gateway configuration to use your own.
+
+## Deployment Instructions
+
+Azure vWAN presents an interesting challenge when deploying using ARM template: after each hub is created, the Routing Service inside will take some time to be fully operational, even if the resource deployment status will be reported as successfully completed from an ARM perspective. This may cause errors using the template if you don't manage the resource deployment dependency chain appropriately. This ARM template has been created to solve this problem and to successfully deploy all intended resources in the template. All resource creations have been carefully chained and made serial, inside the same hub, in order to avoid deployment errors.
+
+Users are encouraged to test and modify the template, but it is recommended to keep VPN resource dependent on the hub, and all additional resources should be dependent on VPN itself: since VPN gateway will require some time to be fully deployed, additional resource deployment will succeed since vWAN Routing Service will have already reached a ready state. A possible alternative is to submit a second time the same deployment, once the error will cause first attempt to fail.
+
+:::image type="content" source="images/azurevirtualwanroutingservicestatefigure2.jpg" alt-text="Routing Service State":::
+
+First attempt of template deployment should fail after 5-6 minutes, then 10-15 minutes will be required for the Routing Services to be in ready state. The second attempt will take longer due to the number of gateways and complexity of configuration. A sample PowerShell script has been provided to automatically manage this retry process in a consistent way, you can find it below. During tests, an average duration of 80-90 minutes has been observed for completion of the entire template deployment, please note that it can varies in your environment.
+
+:::image type="content" source="images/deploymentcompleteinazureportal.jpg" alt-text="Template Deployment State":::
+
+## Successful Deployment
+
+Once the ARM deployment of the template will be completed, you should see something similar to the image below in your Azure Portal:
+
+:::image type="content" source="images/vwanresourcesinazureportal.jpg" alt-text="vWAN Resources in the Azure Portal":::
+
+## PowerShell Helper Script for Deployment
```

Comment:
this will need to be removed from the sample - all samples in the repo must be deployable with the scripts in the repo